addons/website_sale/views/templates.xml

```diff
@@ -181,9 +181,11 @@
     </template>
 
     <template id="products_item" name="Products item">
-        <form action="/shop/cart/update" method="post" class="oe_product_cart h-100 d-flex"
```

Comment:
Wild question: did it work without js to add products to the cart previously ?

Reply:
At some point, but not in the current implementation. We use the same `<a/>` except now it calls the `websiteSale` service to handle the process whereas before it was processed by a mix of `website_sale.js`,`website_sale_utils.js`, website_sale_configurators.js` and `sale_variant_mixin.js` (the latter is still used by the product page though)

Reply:
Well after sole checks it looks like it was working without JavaScript. I and to fix an override of subscription that wasn't trigger on this place before